client/me/help/help-contact/index.jsx

```diff
@@ -260,7 +260,14 @@ module.exports = React.createClass( {
 		}
 
 		if ( ! ( olark.isOlarkReady && sitesInitialized ) ) {
-			return <div className="help-contact__placeholder" />;
+			return (
+				<div className="help-contact__placeholder">
+					<HelpContactForm disabled={true}
+						showSubjectField={ false }
+						onSubmit={ function() {} }
+						buttonLabel={ this.translate( 'Contact us' ) }/>
+				</div>
+			);
 		}
 
 		if ( olark.details.isConversing && olark.isOperatorAvailable ) {
```

Comment:
Ah, thanks. I've removed the subject field (it rendered oddly with the placeholder, and we won't have that for chats anyway). I'm not sure about my treatment of the `onSubmit` prop, which is required and must be a function. This feels a little hacky, and I'm not sure whether there's a better syntax to use here or whether instead I should change the validation (which doesn't feel like the best thing to do).